### PR TITLE
First draft of a help page, plus some command bar cleanup

### LINKED ISSUE #8: Help screen
There should be a help screen, accessible from the command bar, that explains the basic flow (set up match / start match / end match) and documents the keyboard shortcuts.

### PR DESCRIPTION
**Pull request by alan-geller**
Resolves #8.

### CODE REVIEW

File: src/VideoRemise/VideoRemise/HelpPage.xaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                camera.
+                Double-clicking on the video from one camera will expand that video and hide the
+                others; double-clicking again will return to the multi-camera view.
+                </Paragraph>
+                <Paragraph Margin="0,5,0,0">
+                    The command bar buttons are:
+                </Paragraph>
+                <Paragraph Margin="20,0,0,0">
+                    <InlineUIContainer>
+                        <SymbolIcon Symbol="People"/>
```
Comment (atgeller) [inline src/VideoRemise/VideoRemise/HelpPage.xaml:42]: May want to add a name as well as an icon.

File: src/VideoRemise/VideoRemise/HelpPage.xaml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+                    While in the Replaying mode, VideoRemise continues to record the feed and look
+                for triggers. If a trigger occurs while replaying, VideoRemise will stop
+                replaying the previous action and start replaying the new action.
+                </Paragraph>
+                <Paragraph FontSize="18" FontWeight="Bold" Margin="0,5,0,0">Manual Triggers</Paragraph>
+                <Paragraph>
+                    In Recording or Replaying mode, the following keys may be used to trigger the system:
+                </Paragraph>
+                <Paragraph Margin="20,0,0,0">
+                    <Bold>Return</Bold> signals a Halt with no lights; for instance, the referee has called Halt
```
Comment (atgeller) [inline src/VideoRemise/VideoRemise/HelpPage.xaml:96]: May want to use enter/return or something like that, not sure how many people will know return=enter.

**Review by atgeller — DISMISSED** [review]

File: src/VideoRemise/VideoRemise/HelpPage.xaml (hunk 1 above)
Comment (alan-geller) [reply src/VideoRemise/VideoRemise/HelpPage.xaml:42]: Done.

File: src/VideoRemise/VideoRemise/HelpPage.xaml (hunk 2 above)
Comment (alan-geller) [reply src/VideoRemise/VideoRemise/HelpPage.xaml]: Done.

**Pull request merged**